src/main/java/digital/slovensko/autogram/core/eforms/UpvsEFormResources.java

```diff
@@ -73,4 +74,16 @@ public boolean findResources() throws XMLValidationException {
 
         return true;
     }
+
+    private byte[] getRemoteResource(String url) {
+        byte[] bytes = getResource(url);
```

Comment:
pls, typy vyhoďme všade, kde sa dá. Ten `var` máme veľmi radi.

Reply:
jasné, musím si dať na to pozor, nie som na to vôbec zvyknutý 😅